packages/bierzo-wallet/src/utils/test/e2e.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
We also can override [eslint rules](https://eslint.org/docs/user-guide/configuring#example-configuration) for our tests.

Reply:
Yeah, right. In this case I followed the same pattern that we have so far. Let's think about updating this later on.